src/ESPEasy-Globals.h

```diff
@@ -988,6 +989,7 @@ struct DeviceStruct
   boolean GlobalSyncOption;
   boolean TimerOption;
   boolean TimerOptional;
+  boolean ResetPin;  
```

Comment:
You may not change the order of parameters in these structs.
The structs will be stored in the settings file as a binary dump of the settings struct. If you change the order, the existing settings will no longer work.